collect_app/src/main/res/xml/preferences.xml

```diff
@@ -75,6 +75,12 @@
             android:key="font_size"
             android:title="@string/font_size" />
         <CheckBoxPreference
+            android:id="@+id/instance_sync"
+            android:defaultValue="true"
+            android:key="instance_sync"
```

Comment:
Yeah, but the value for the in the PreferenceKey.java can use the value from the resource, right?

public static final String KEY_INSTANCE_SYNC = Collect.getInstance().getString(R.string.key_instance_sync);

Or am I overthinking this? What do you think @lognaturel?